src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/MessageId.kt

```diff
@@ -75,13 +78,121 @@ data class MessageId(
         fun build(): MessageId
     }
 
-    fun toBuilder(): Builder = AutoBuilder_MessageId_Builder(this)
+    fun toBuilder(): Builder = MessageIdBuilderImpl(this)
 
     companion object {
         @JvmStatic
         val DEFAULT: MessageId = MessageId("", Direction.OUTGOING, 0, Instant.EPOCH)
 
         @JvmStatic
-        fun builder(): Builder = AutoBuilder_MessageId_Builder()
+        fun builder(): Builder = MessageIdBuilderImpl()
     }
-}
\ No newline at end of file
+}
+
+
+private const val SEQUENCE_NOT_SET = -1L
```

Comment:
Why we add this constant?

Reply:
Because I want to have `long` in the builder class for sequence instead of `Long`. So, we need to have an indicator that sequence is not set. The `0` looks like a valid value so I used `-1` as a marker

Reply:
Maybe use minimal long instead. 
I could be wrong, but I saw negative number in the sequence field in an act logs in a client project solution. I think that users can use this approach to trace messages before sending.